cmd/app.go

```diff
@@ -1105,15 +1140,36 @@ func processAnswers(
 		}
 	}
 
+	// answers to questions will be added to map
 	if interactive {
-		// answers to questions will be added to map
-		err := askQuestions(tv, answers)
-		if err != nil {
-			return answers, err
+		if ctx.String("values") != "" {
```

Comment:
This looks like duplicate logic of the block starting at L1101 to me, are we not askQuestions twice?

Reply:
removed duplicate logic